packages/x-adapter/src/mappers/combine-mappers.ts

```diff
@@ -0,0 +1,29 @@
+import { deepMerge } from '@empathyco/x-deep-merge';
+import { DeepPartial } from '@empathyco/x-utils';
+import { Mapper } from '../types/mapper.types';
+
+/**
+ * Combines the given {@link Mapper | mappers} into a single {@link Mapper | mapper function}.
+ *
+ * @param mappers - The {@link Mapper | mapper functions} to combine.
+ * @returns The combined {@link Mapper | mapper function}.
+ *
+ * @public
+ */
+export function combineMappers<From, To>(
+  ...mappers: Mapper<From, DeepPartial<To>>[]
+): Mapper<From, To> {
+  return mappers.length === 1
+    ? (mappers[0] as Mapper<From, To>)
+    : (from, context) => {
+        if (!context.to) {
+          context.to = {};
```

Comment:
I understand the use of this `to` parameter in the context but the name is confusing.

Maybe something more clear like: `previousMappedValue`,  `previousValue`, `preValue`...

Reply:
We used the `to` parameter because it is defined in the `MapperContext` interface (packages/x-adapter/src/types/mapper.types.ts) and made sense to us to use it because of its documentation. If you think that another name fits better we can either rename it or just use a new name as the interface allows it.

Reply:
Yes, we can rename it in the `MapperContext`

Reply:
Renamed from `to` to `mappedValue`